config/database.yml.sample

```diff
@@ -3,7 +3,7 @@ defaults: &defaults
   encoding: <%= ENV['DB_ADAPTER'] == "mysql2" ? "utf8mb4" : "" %>
   username: <%= ENV["DB_ADAPTER"] == "postgresql" ? 'postgres' : '' %>
   database: roadmap_<%= ENV['RAILS_ENV'] %>
-  pool: 5
+  pool: 16
```

Comment:
Is this for the parallel gem?

Reply:
yeah, essentially each thread/process takes an additional db connection, so it was flooding the default 5.  16 is a bit overkill for development, but it means we don't end up with connection-timeout errors